packages/client/package.json

```diff
@@ -108,5 +109,6 @@
 		"eslint-plugin-import": "2.26.0",
 		"eslint-plugin-vue": "9.2.0",
 		"start-server-and-test": "1.14.0"
-	}
+	},
+	"packageManager": "yarn@3.2.1"
```

Comment:
```suggestion
	}
```

Reply:
Sorry about that, yarn did it and i didn't catch that